example_workspace/.hlasmplugin/proc_grps.json

```diff
@@ -1,10 +1,14 @@
 {
   "pgroups": [
     {
-      "name": "P1",
-      "libs": [
-        "libs","libs2"
-      ]
+       "name": "P1",
+       "libs": [
+          "libs","libs2"
+        ],
+        "asm_options": {
+            "SYSPARM": "SEVEN",
+            "PROFILE": "MAC1"
```

Comment:
```suggestion
```
I would remove the PROFILE option, since it is not implemented yet. And even when it is implemented, we do not need to showcase every single feature of the plugin in `example_workspace`... A PROFILE option set for all the simple example sources could be very confusing for newcomers.